proposed/http-factory/http-factory.md

```diff
@@ -34,7 +34,10 @@ in separate classes.
 Has the ability to create client requests.
 
 ```php
-namespace Psr\Http\Message;
+namespace Psr\Http\Factory;
```

Comment:
Good question... I'm not really sure where that came from, I think it's just because http-interop didn't share the namespace.